scripts/globals/mobskills/Bombilation.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Bombilation
+--
+--  Description: Resets TP of all targets in an area of effect.
+--  Type: Enfeebling
+--  Ignores Shadows
+--  Range: Unknown radial
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+   local reset = 0;
```

Comment:
> Nit-picky is cool. Can I attempt to fix in the AM though? :)

There's no rush 😃 I'm just an insomniac with time to read pull requests and issues tonight